tests/gsfpy3_08/conftest.py

```diff
@@ -0,0 +1,92 @@
+import shutil
+from dataclasses import dataclass
+from enum import Enum, auto
+from pathlib import Path
+
+import pytest
+from pytest_cases import fixture_union
+
+
+class GsfVersion(Enum):
+    V03_06 = auto()
+    V03_07 = auto()
+    V03_08 = auto()
+
+
+@dataclass(frozen=True)
+class GsfDatafile:
+    gsf_version: GsfVersion
+    path: Path
+    num_beams: int
+
+
+GSF_03_06_DATAFILE = GsfDatafile(
+    GsfVersion.V03_06,
+    Path(__file__).parent.parent
+    / "test_data"  # noqa: W503
+    / "0029_20160323_185603_EX1604_MB.gsf.mb121",  # noqa: W503
+    num_beams=432,
+)
+
+GSF_03_07_DATAFILE = GsfDatafile(
+    GsfVersion.V03_07,
+    Path(__file__).parent.parent
+    / "test_data"  # noqa: W503
+    / "0059_20181102_212138_EX1811_MB_EM302.gsf.mb121",  # noqa: W503
+    num_beams=432,
+)
+
+# GSF_03_08_DATAFILE = GsfDatafile(
+#     GsfVersion.V03_08,
+#     Path(__file__).parent.parent
+#     / "test_data"
+#     / "0059_20181102_212138_EX1811_MB_EM302.gsf.mb121",
+#     num_beams=432,
+# )
```

Comment:
commented data?

Reply:
Fixed - the GSF 3.08 tests now use only GSF_03_08_DATAFILE test data